markdown/dev/tutorials/getting-started-vercel/en.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Getting started with Vercel
+order: 14
+---
+
+## What is Vercel?
+
+Vercel is an online service that builds and deploys web apps and
+websites.
+It allows you to preview designs you are working on and changes
+you are making to the FreeSewing app and websites.
+By sharing links to your builds/deployments, other people can also
+preview your work.
+
+Vercel offers a free "Hobby" account for personal, non-commercial use,
+so everyone is able to use it.
+
+<Note compact>
+This tutorial assumes that you already have a GitHub account
+containing a fork of the FreeSewing repository.
+</Note>
```

Comment:
I'd like at least a sentence here detailing why a freesewing developer would find a personal vercel setup desirable. I think all of our other "getting started" tutorials that aren't for operating systems explain why you might want to use the given platform, and I think it's an essential part of the documentation.

I think it'd also be helpful to explain that we use vercel to preview pull requests, so if you intend to always submit your work to the main repository, it isn't necessary to have your own vercel account to share your work.

Reply:
Added both suggestions by adding next section.